bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,17 @@
+def bubble_sort(a_list):
+    for n in range(len(a_list)-1,0,-1):
+        for k in range(n):
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpuo0eqhat/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpuo0eqhat/bubble_sort/Python/bubble_sort.py
@@ -7,6 +7,7 @@
                 a_list[k+1] = temp
     return a_list
 
+
 def main():
     arr = [2, 3, 0, 4]
     sorted_arr = bubble_sort(arr)
```

Reply:
I'm a little confused what the issue was? >.<